### PR TITLE
ARUHA-906: Fix wrong formatting in swagger

### PR DESCRIPTION
**Pull request by SergKam**
ARUHA-906: Merge Documentation
https://techjira.zalando.net/browse/ARUHA-906

## Description
Tha parser was failing to parse this part of swagger in CursorDistanceResult.
The ident level was wrong.

## Review
- [x] Tests
- [x] Documentation
- [x] CHANGELOG

### CODE REVIEW

**Comment by lmontrieux** [comment]
👍

**Comment by SergKam** [comment]
:+1:

File: api/nakadi-event-bus-api.yaml [hunk 1]
```diff
@@ -1947,16 +1947,16 @@ definitions:
...
+      - type: Object        
+        properties:
+          distance:
+            type: number
+            format: int64
+            description: |
+              Number of events between two offsets. Initial offset is exclusive. It's only zero when both provided offsets
+              are equal.
+        required:
+          - distance
```
Comment (v-stepanov) [inline api/nakadi-event-bus-api.yaml]: what is the purpose of these changes? For me the original formatting looks pretty good and swagger-editor doesn't complain as well.

Comment (SergKam) [inline api/nakadi-event-bus-api.yaml]: the ident level is wrong. allOf should contain a list of objects. In this case one is a referense and another is the object defined here. Although the yaml structure is correct(the editor will not complain) the semantical meaning was totally wrong. Any program that try to generate this object will fail or ignore unrecognised field "properties" at the same level as allOff.

Comment (v-stepanov) [inline api/nakadi-event-bus-api.yaml]: ah, ok. Then good :)

**Comment by v-stepanov** [comment]
👍

**Comment by SergKam** [comment]
@v-stepanov @lmontrieux Sorry I need another approval. It should be "object" not "Object".

**Comment by SergKam** [comment]
:+1:

**Comment by v-stepanov** [comment]
👍
